### DISCUSSION TITLE
Question about function "fluxreg_reflux" in AmrCore

### DISCUSSION THREAD

**Discussion by wjwhit** [AMReX-Codes/amrex#3213] [Q&A] [answered]
When I read the function fluxreg_reflux in AMReX_FluxReg_1D_C.H, I was confused by the face.isLow(). I think face.isLow() returns "true" when at face i-1/2. Then, when updating s, why the code uses flux f(i+1), which is at face i+1/2. The same in AMReX_FluxReg_2D_C.H and AMReX_FluxReg_3D_C.H. I included a snippet of the code below. Thanks! 

`if (face.isLow()) {
        for (int n = 0; n < ncomp; ++n) {
            for (int i = lo.x; i <= hi.x; ++i) {
                s(i,0,0,n+scomp) += -mult*f(i+1,0,0,n)/v(i,0,0);
            }
        }
    } else {
        for (int n = 0; n < ncomp; ++n) {
            for (int i = lo.x; i <= hi.x; ++i) {
                s(i,0,0,n+scomp) += mult*f(i,0,0,n)/v(i,0,0);
            }
        }
    }`

**Comment by WeiqunZhang** [comment, marked as answer]
`face.isLow()` tells us if a coarse/fine interface is low relative to a *fine* Box.  To the coarse cell that receives the correction, it actually comes the high side.
